docs/index.rst

```diff
@@ -18,10 +18,22 @@ communication between teams. Head over to the :ref:`intro` to learn more, or jum
    :maxdepth: 2
 
    intro
-   guides
-   reference
-   community
-   contributing
+
+.. toctree::
+   :maxdepth: 2
+
+   /guides/tutorials
+   /guides/workflows_patterns
+   /guides/how_to_guides
+
+
+.. toctree::
```

Comment:
Two things:
1. Can you explain the change?
2. I'm super confused, I built this locally and still see `community` and `contributing` show up here.

Reply:
Well, there's "explain the change" and "explain the change"....apparently, this kind of TOC collapsing is a fairly common issue with sphinx themes (?) and there are multiple approaches. With ONE `toctree` you only get ONE section that can be opened at a time. If we break it out like in the experiment here (I'll do it for all of them now that the build shows it works), you can have multiple open at the same time.

Reply:
...but WHY it is built like that...no idea.

Reply:
(at least, I think...I was viewing this as a guess-and-check exercise)

Reply:
Oooh okay, that makes sense!

Reply:
Makes sense but also, 🤯 just a little.